app/cdap/components/DirectiveInput/Components/AutoComplete/styles.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { makeStyles } from '@material-ui/core';
+
+export const useStyles = makeStyles({
+  resultRow: {
+    padding: 10,
+    borderBottom: '1px solid #E0E0E0',
+    '&:hover': {
+      background: '#EFF0F2',
+      cursor: 'pointer',
+    },
+  },
+  activeRow: {
+    background: '#EFF0F2',
+  },
+  directiveTitle: {
+    fontFamily: 'Noto Sans',
+    fontStyle: 'normal',
+    fontWeight: 600,
+    fontSize: '16px',
+    letterSpacing: '0.15px',
+    color: '#616161',
```

Comment:
use number directly instead of using number with px..and use MUI colors ..check the entire file for changes

Reply:
Done